docker-compose-ci.yml

```diff
@@ -0,0 +1,48 @@
+version: "3.3"
+services:
+  node_0:
+    container_name: node_0
+    image: services-node:${TAG}
+    environment:
+      - NODE_ID=0
+    volumes:
+      - "./hedera-node/configuration/compose:/opt/hedera/services/config-mount"
+      - "./compose-network/node0/saved:/opt/hedera/services/data/saved"
+      - "./compose-network/node0/output:/opt/hedera/services/output"
+    ports:
+      - "50211:50211"
+
+  node_1:
+    container_name: node_1
+    image: services-node:${TAG}
+    environment:
+      - NODE_ID=1
+    volumes:
+      - "./hedera-node/configuration/compose:/opt/hedera/services/config-mount"
+      - "./compose-network/node1/saved:/opt/hedera/services/data/saved"
+      - "./compose-network/node1/output:/opt/hedera/services/output"
+    ports:
+      - "50212:50211"
+
+  node_2:
+    container_name: node_2
+    image: services-node:${TAG}
+    environment:
+      - NODE_ID=2
+    volumes:
+      - "./hedera-node/configuration/compose:/opt/hedera/services/config-mount"
+      - "./compose-network/node2/saved:/opt/hedera/services/data/saved"
+      - "./compose-network/node2/output:/opt/hedera/services/output"
+    ports:
+      - "50213:50211"
+
+  postgres:
```

Comment:
Again, shouldn't need postgres. Definitely shouldn't need it both in CI and in compose file.

Reply:
Resolving this, since we removed the docker-compose-ci.yml.